keboliu/sonic-buildimage
# Install smartmontools with apt-get and upgrade it to 7.2-1

Signed-off-by: Kebo Liu <kebol@nvidia.com>



#### Why I did it
Smartmontools 6.6 has issue with read SMART info of nvme SSD
Smartmontools can be installed with apt-get, no need to build and install

#### How I did it
Use apt-get to install smartmontools 7.2-1
Remove previous make files for smartmontools 6.6

#### How to verify it
verify with "smartctl" can read out correct SMART info on NVME ssd.
verify "show platform ssdhealth" can still work

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
